image/manifest.go

```diff
@@ -87,7 +87,22 @@ func (m *manifest) validate(w walker) error {
 	return nil
 }
 
-func (m *manifest) unpack(w walker, dest string) error {
+func (m *manifest) unpack(w walker, dest string) (retErr error) {
+	// error out if the dest directory is not empty
+	if s, err := ioutil.ReadDir(dest); err != nil && !os.IsNotExist(err) || len(s) > 0 {
+		if err != nil {
+			return fmt.Errorf("failed to open %s: %v", dest, err)
+		}
+		return fmt.Errorf("%s is not empty", dest)
```

Comment:
@wking  I just want the if statement in a single block, but I'm ok with your change